README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
-   You should see `C:\ProgramData\chocolatey\bin` in the path.
+    ```bash
+    mkdir <directory-name>
+    ```
 
-4. Close and reopen the command prompt before using `choco` commands.
+1.  Follow the [Generating a new SSH](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) instructions.
 
-### Install Ruby package
+### Clone and build the DevDocs repository
 
-1. Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+You may have to close and reopen the Git Bash application after the Choco installations.
 
-1. Install the ruby package:
+1.  Open Git Bash. The **Git Bash** executable is on the shortcut menu.
 
-   ```shell
-   > choco install ruby --version 2.5.3.101
-   ```
+1.  Change to the directory you created for Git repositories and clone the DevDocs repository.
 
-    Reopen the Command prompt to enable the installation.
+    ```bash
+    git clone git@github.com:magento/devdocs.git
+    ```
 
-> **NOTE**  
-> If you encounter problems with Ruby, or the `gem` command is not recognized, you can install the `rubyinstaller-devkit.exe` development kit located in the `c:\ProgramData\chocolatey\bin` folder.
+1.  Change to the `devdocs/` directory.
 
-1. Install the Ruby development kit
+1.  Install [Bundler](https://bundler.io).
 
-   ```shell
-   ridk install
-   1 - MSYS2 base installation
-   2 - MSYS2 system update (optional)
-   3 - MSYS2 and MINGW development toolchain
+    ```bash
+    gem install bundle
+    ```
 
-   Which components shall be installed? If unsure press ENTER [1,2,3]
-   ```
+1.  Install the project dependencies using Bundler.
 
-   Press *Enter*. Same for similar requests until the installation is complete.
+    ```bash
+    bundle install
+    ```
 
-### Clone and build the DevDocs repository
+1.  Build the site.
+
+    ```bash
+    bundle exec jekyll serve
+    ```
 
-Refer to [Installing the project dependencies locally](#installing-the-project-dependencies-locally).
+    ```terminal
+    Configuration file: C:/Users/Administrator/mage/devdocs/_config.yml
+                Source: C:/Users/Administrator/mage/devdocs
+           Destination: C:/Users/Administrator/mage/devdocs/_site
+     Incremental build: disabled. Enable with --incremental
+          Generating...
+                        done in 643.551 seconds.
+     Auto-regeneration: enabled for 'C:/Users/Administrator/mage/devdocs'
+        Server address: http://127.0.0.1:4000/
+      Server running... press ctrl-c to stop.
+    ```
 
-> **NOTE**  
-> You can create and manage the `.bash_profile` file using Git Bash, which is useful for aliases and customization. This file is in the `users/Administrator` directory.
+>  **NOTE**  
+>  The `.bash_profile` file CAN be created and managed using Git Bash, which is useful for aliases and other customizations. This file is in the `users/Administrator` folder.
```

Comment:
The note about the `.bash_profile`seems confusing.  I am not sure if the Git Bash installation creates the .bash_profile for me, or if I need to create it and put it in the `users/Administrator` folder.  
Maybe:
If you want to add aliases and other customizations for your environment,  use the `.bash_profile` file in the `users/Administrator` folder.

I am going to install this on my Windows at home, so I'll find out :-).

Reply:
I agree with @meker12 
`.bash_profile` can be useful only for a person who works on Mac. If the user works on Windows, it is very likely that the user is not aware of `.bash_profile`.

Reply:
This is informational and can be very useful. Think of it as a tip.

Reply:
This information requires from Windows user to have extra knowledge that Windows users commonly don't have.